devtools/server/actors/replay/auth.js

```diff
@@ -129,11 +129,9 @@ function notifyWebChannelTarget(channel, target) {
 
 function handleAuthChannelMessage(channel, _id, message, target) {
   const { type } = message;
-  // TODO [ryanjduffy]: Add support for app login to use the browser auth flow
-  // by extending this logic to support a webchannel message from the client
-  // (the app) to request a login which would launch the sign in page in the
-  // user's preferred browser.
```

Comment:
Out of curiosity, what happens if a page sends this message but the user is still on an old build?